### DISCUSSION TITLE
UC detected by datadome

### DISCUSSION THREAD

**Discussion by JohnDoe15152** [ultrafunkamsterdam/undetected-chromedriver#1229] [Q&A]
Hello everyone,
I currently use UC to access this site: https://www.intermarche.com. This site is provided with datadome protection. And despite all my efforts, I can't access it with uc without triggering the datadome captcha. I don't use any particular option, just the basic code, without headless mode. I tried to use several options, but without success. Here is my current code: 

`options = uc.ChromeOptions()`
`driver = uc.Chrome(options=options, version_main=112)`
`url ="https://www.intermarche.com/"`
`driver.get(url)`

How to bypass the datadome captcha please? Maybe @ultrafunkamsterdam please? UC is however supposed to bypass datadome, I don't understand. Thanks for the help!

**Comment by damodharheadrun** [comment]
@JohnDoe15152  Please use the latest version of UC and chromedriver version should be always upto-date

**Reply by damodharheadrun** [reply]
We have to check in the normal browser( if the normal human browser is blocked we can proceed with the crawling)
